gramps/plugins/rel/rel_nl.py

```diff
@@ -69,9 +70,207 @@
 _nibling_level = [ "", "",
     "achter", "achterachter", "achterachterachter" ]
 
+_parents_level = ["",
+                   "ouders",
+                   "grootouders",
+                   "overgrootouders",
+                   "betovergrootouders",
+                   "overgrootouders 5e graad",
+                   "overgrootouders 6e graad",
+                   "overgrootouders 7e graad",
+                   "overgrootouders 8e graad",
+                   "overgrootouders 9e graad",
+                   "overgrootouders 10e graad",
+                   "overgrootouders 11e graad",
+                   "overgrootouders 12e graad",
+                   "overgrootouders 13e graad",
+                   "overgrootouders 14e graad",
+                   "overgrootouders 15e graad",
+                   "overgrootouders 16e graad",
+                   "overgrootouders 17e graad",
+                   "overgrootouders 18e graad",
+                   "overgrootouders 19e graad",
+                   "overgrootouders 20e graad",
+                   "overgrootouders 21e graad",
+                   "overgrootouders 22e graad",
+                   "overgrootouders 23e graad",
+                   "overgrootouders 24e graad",
+                   "overgrootouders 25e graad",
+                   "overgrootouders 26e graad",
+                   "overgrootouders 27e graad",
+                   "overgrootouders 28e graad",
+                   "overgrootouders 29e graad",
+                   "overgrootouders 30e graad",
+                   "overgrootouders 31e graad",
+                   "overgrootouders 32e graad",
+                   "overgrootouders 33e graad",
+                   "overgrootouders 34e graad",
+                   "overgrootouders 35e graad",
+                   "overgrootouders 36e graad",
+                   "overgrootouders 37e graad",
+                   "overgrootouders 38e graad",
+                   "overgrootouders 39e graad",
+                   "overgrootouders 40e graad",
+                   "overgrootouders 41e graad",
+                   "overgrootouders 42e graad",
+                   "overgrootouders 43e graad",
+                   "overgrootouders 44e graad",
+                   "overgrootouders 45e graad",
+                   "overgrootouders 46e graad",
+                   "overgrootouders 47e graad",
+                   "overgrootouders 48e graad",
+                   "overgrootouders 49e graad",
+                   "overgrootouders 50e graad", ]
+
+_siblings_level = ["",
+                   "broers en zussen",
+                   "ooms/tantes",
```

Comment:
No "en" instead of a slash, like with "broers en zussen"?

Reply:
Good idea, let's change it into:
"ooms en tantes",

Reply:
Do you want the whole list changed in the same way?

"oudooms/tantes" -> "oudooms en tantes"
"overoudooms/tantes" -> "overoudooms en tantes"
"overoudooms/tantes 5e graad" -> "overoudooms en tantes 5e graad"
etc...

Reply:
This is wrong, because it should be oudooms en -tantes, where the dash means that oud as a prefix is left out.

Reply:
So you are suggesting:

"oudooms/tantes" -> "oudooms en -tantes"
"overoudooms/tantes" -> "overoudooms en -tantes"
"overoudooms/tantes 5e graad" -> "overoudooms en -tantes 5e graad"

and further down:

"achterneven/nichten" -> "achterneven en -nichten"
"achterneven/nichten 4e graad" -> "achterneven en -nichten 4e graad"

Is that correct?

Reply:
Good suggestion! That looks correct.

Reply:
Yes, that's correct.

Reply:
Yes, I agree. Thanks all for good help!!